src/main/java/CR7.java

```diff
@@ -0,0 +1,60 @@
+import commands.*;
+import functions.*;
```

Comment:
Perhaps you could reduce the usage of wildcard('`*`') when importing classes, it's better to have multiple-line import statements. You could configure some IDE settings to allow this and let it write the import for you. :blush: